app/serializers/user.ts

```diff
@@ -1,6 +1,28 @@
+import DS from 'ember-data';
+import config from 'ember-get-config';
+import { Resource } from 'osf-api';
 import OsfSerializer from './osf-serializer';
 
+const {
+    OSF: {
+        apiUrl,
+        apiNamespace,
+    },
+} = config;
+
 export default class UserSerializer extends OsfSerializer {
+    normalize(modelClass: DS.Model, resourceHash: Resource) {
+        const result = super.normalize(modelClass, resourceHash);
+        // Manually insert a `sparseNodes` relationship to the model because the API doesn't serialize this relationship
+        result.data.relationships!.sparseNodes = {
+            links: {
+                related: {
+                    href: `${apiUrl}/${apiNamespace}/sparse/users/${resourceHash.id}/nodes`,
+                },
+            },
+        };
```

Comment:
This should be conditional because there is a plan to add this relationship, eventually.

Reply:
Will do